### ISSUE TITLE
The error thrown by client.messages.create seems to be a string

### ISSUE THREAD

**Issue by Mahan-F** [plivo/plivo-node#183]
I am having an issue where when I try to catch the errors thrown by the `client.messages.create()` function seem to be a string.

I am having an error which says:
```
{"api_id":"hidden","error":"'************' is not a valid phone number."}
```

This JSON can be seen when I log the error in a catch block but the JSON is right after the stack trace. Usually the JSON can be accessed separately but when I logged `typeof error` it returns string which means there is no good way for me to access this JSON in order to return a meaningful message.

Is this a bug or am I doing something wrong here?

**Comment by narayana-plivo** [comment]
@Mahan-F  As per code, if any response status code is greater than 200 and less than 300, return the error message in string format.  The JSON.stringify() will convert to string and return it.

https://github.com/plivo/plivo-node/blob/master/lib/rest/request.js#L172 

@nixonsam  Please confirm the behaviour.  its common for voice and sms. If u need more detail on, ping me
@huzaif-plivo @abhineet-plivo

**Comment by huzaif-plivo** [comment]
@Mahan-F This has been addressed in our latest release([v4.17.0](https://github.com/plivo/plivo-node/tree/v4.17.0)). If you have any more inquiries, please feel free to contact our [support team](https://support.plivo.com/hc/en-us).
